deisctl/cmd/cmd.go

```diff
@@ -405,7 +405,7 @@ func Config(target string, action string, key []string) error {
 // RefreshUnits overwrites local unit files with those requested.
 // Downloading from the Deis project GitHub URL by tag or SHA is the only mechanism
 // currently supported.
-func RefreshUnits(dir string, tag string) error {
+func RefreshUnits(dir string, tag string, get func(string) (*http.Response, error)) error {
```

Comment:
Instead of making a custom `get` function, you could simply run a [httptest](http://golang.org/pkg/net/http/httptest/) server that hosts the units and refactor out the hardcoded rootURL. I think that would be a cleaner approach. What do you think about that?